### ISSUE TITLE
Tests fail while building the Debian package

### ISSUE THREAD

**Issue by fmarier** [letsencrypt/letsencrypt#456]
As part of the Debian package build, the tests are run. Unfortunately, they currently fail:

```
======================================================================
ERROR: test_plugins (letsencrypt.tests.cli_test.CLITest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/cli_test.py", line 44, in test_plugins
    self._call(['plugins',] + list(args))
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/cli_test.py", line 18, in _call
    ret = cli.main(args)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/cli.py", line 390, in main
    return args.func(args, config, plugins)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/cli.py", line 200, in plugins_cmd
    verified.prepare()
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/plugins/disco.py", line 187, in prepare
    return [plugin_ep.prepare() for plugin_ep in self._plugins.itervalues()]
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/plugins/disco.py", line 91, in prepare
    self._initialized.prepare()
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt_apache/configurator.py", line 147, in prepare
    temp_install(self.conf('mod-ssl-conf'))
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt_apache/configurator.py", line 1170, in temp_install
    shutil.copyfile(constants.MOD_SSL_CONF, options_ssl)
  File "/usr/lib/python2.7/shutil.py", line 83, in copyfile
    with open(dst, 'wb') as fdst:
IOError: [Errno 13] Permission denied: '/etc/letsencrypt/options-ssl.conf'

======================================================================
ERROR: test_name1_dvsni1 (letsencrypt.tests.auth_handler_test.GetAuthorizationsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mock.py", line 1210, in patched
    return func(*args, **keywargs)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/auth_handler_test.py", line 107, in test_name1_dvsni1
    authzr = self.handler.get_authorizations(["0"])
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/auth_handler.py", line 79, in get_authorizations
    self._respond(cont_resp, dv_resp, best_effort)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/auth_handler.py", line 137, in _respond
    self._poll_challenges(chall_update, best_effort)
  File "/usr/lib/python2.7/dist-packages/mock.py", line 964, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/mock.py", line 1027, in _mock_call
    ret_val = effect(*args, **kwargs)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/auth_handler_test.py", line 167, in _validate_all
    azr.body.combinations)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/acme_util.py", line 131, in gen_authzr
    "expires": datetime.datetime(now.year, now.month + 1, now.day),
ValueError: day is out of range for month

======================================================================
ERROR: test_name3_dvsni3_rectok_3 (letsencrypt.tests.auth_handler_test.GetAuthorizationsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/mock.py", line 1210, in patched
    return func(*args, **keywargs)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/auth_handler_test.py", line 131, in test_name3_dvsni3_rectok_3
    authzr = self.handler.get_authorizations(["0", "1", "2"])
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/auth_handler.py", line 79, in get_authorizations
    self._respond(cont_resp, dv_resp, best_effort)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/auth_handler.py", line 137, in _respond
    self._poll_challenges(chall_update, best_effort)
  File "/usr/lib/python2.7/dist-packages/mock.py", line 964, in __call__
    return _mock_self._mock_call(*args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/mock.py", line 1027, in _mock_call
    ret_val = effect(*args, **kwargs)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/auth_handler_test.py", line 167, in _validate_all
    azr.body.combinations)
  File "/home/francois/devel/remote/letsencrypt/build-area/letsencrypt-0.20150531/letsencrypt/tests/acme_util.py", line 131, in gen_authzr
    "expires": datetime.datetime(now.year, now.month + 1, now.day),
ValueError: day is out of range for month

----------------------------------------------------------------------
Ran 360 tests in 3.178s

FAILED (errors=3)
```

**Comment by kuba** [comment]
`test_plugins` is fixed in #440, nasty time bug in #467

**Comment by kuba** [comment]
This is OBE, please close
